tests/test_utils.py

```diff
@@ -87,6 +89,44 @@ def testPatchCorners(self):
             np.testing.assert_array_almost_equal(getPatchCorners(tractInfo, patchId), self.patchCorners[i])
 
 
+class TestHttpSessionAdapters(lsst.utils.tests.TestCase):
```

Comment:
A few comments added to the test to explain your intentions / the code flow to future readers would be nice.